lib/ansible/playbook/play_context.py

```diff
@@ -216,6 +217,9 @@ class PlayContext(Base):
     _gather_timeout   = FieldAttribute(isa='string', default=C.DEFAULT_GATHER_TIMEOUT)
     _fact_path        = FieldAttribute(isa='string', default=C.DEFAULT_FACT_PATH)
 
+    # Module defaults
+    _module_defaults  = FieldAttribute(isa='dict')
```

Comment:
Use ```default={}``` here so that you don't have to compare against None in the rest of the code (unintuitive to a python programmer but I checked the FieldAttribute code and it does a deepcopy(default) in this case to make it work the right way).